torch/_dynamo/symbolic_convert.py

```diff
@@ -1460,6 +1494,94 @@ def MATCH_KEYS(self, inst):
     INPLACE_XOR = stack_op(operator.ixor)
     INPLACE_OR = stack_op(operator.ior)
 
+    # 3.11 opcodes
+    def RESUME(self, inst):
+        pass
+
+    def BINARY_OP(self, inst):
+        opname = dis._nb_ops[inst.arg][0][3:]
+        if opname.startswith("INPLACE"):
+            return getattr(self, "INPLACE_" + opname[8:])(inst)
+        return getattr(self, "BINARY_" + opname)(inst)
+
+    def PRECALL(self, inst):
+        pass
+
+    def KW_NAMES(self, inst):
+        kw_names = self.code_options["co_consts"][inst.arg]
+        assert isinstance(kw_names, tuple)
+        for name in kw_names:
+            assert isinstance(name, str)
+        assert self.kw_names is None
+        self.kw_names = ConstantVariable(value=kw_names)
+
+    def PUSH_NULL(self, inst):
+        self.push(NullVariable())
+
+    # TODO is push=1 correct?
```

Comment:
Not sure actually - consider leaving this off and wrapping it in a custom one...